util/grpc/errors_test.go

```diff
@@ -113,7 +113,7 @@ func Test_kubeErrToGRPC(t *testing.T) {
 				grpcStatus := status.New(codes.Unavailable, err.Error())
 				return grpcStatus.Err()
 			},
-			expectedGRPCStatus: status.New(codes.Unavailable, apierr.NewServerTimeout(schema.GroupResource{}, "update", 1).Error()),
+			expectedGRPCStatus: status.New(codes.Unavailable, "Command timed out before the specified conditions were met"),
```

Comment:
After running this locally, it should be like this:

```suggestion
			expectedGRPCStatus: status.New(codes.Unavailable, apierr.NewServerTimeout(schema.GroupResource{}, "update", 1).Error()),
		},
		{
			name: "will return DeadlineExceeded if apierr.IsTimeout",
			givenErrFn: func() error {
				return apierr.NewTimeoutError("context exceeded", 1)
			},
			expectedErrFn: func() error {
				grpcStatus := status.New(codes.DeadlineExceeded, "Command timed out before the specified conditions were met")
				return grpcStatus.Err()
			},
			expectedGRPCStatus: status.New(codes.DeadlineExceeded, "Command timed out before the specified conditions were met"),
		},
```

Reply:
Oh, I get it now!
Thanks for helping me understand.